README.md

```diff
@@ -1,7 +1,12 @@
-# Knapsack
+> [!WARNING]
+> Knapsack is [archived](https://knapsackpro.com/knapsack_gem?utm_source=github&utm_medium=readme&utm_campaign=knapsack_gem_archived&utm_content=warning_knapsack_gem). But [Knapsack Pro](https://knapsackpro.com?utm_source=github&utm_medium=readme&utm_campaign=knapsack_gem_archived&utm_content=warning_knapsack_pro) is available.
```

Comment:
Perhaps a link to migration steps (`MIGRATE_TO_KNAPSACK_PRO.md`) would be helpful here and easier to find than down below.